hack/release.sh

```diff
@@ -1,224 +0,0 @@
-#!/bin/bash
```

Comment:
This would fail now anyway. Let's directly switch to image promotion when we merge this PR